### PR TITLE
Conformance results for v1.23/asuscloudinfra

### PR DESCRIPTION
**Pull request by IrisChanTW**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [x] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] If your product/project is open source, did you include the `repo_url`?
* [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

### CODE REVIEW

File: v1.23/asuscloudinfra/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+vendor: ASUS Cloud
+name: ASUS Cloud Infra 
+version: 4.0.0
+website_url: www.asuscloud.com
```
Comment (Riaankl) [inline v1.23/asuscloudinfra/PRODUCT.yaml:4]: can you please add '[http://](http:)' or '[https://](https://github.com/cncf/k8s-conformance/pull/1849/files)'to the URL to help the Bot to check it.

**Comment by taylorwaggoner** [comment]
@IrisChanTW please see comment above, thanks

**Comment by IrisChanTW** [comment]
Hi,

Please help to confirm the PR#1849
<https://github.com/cncf/k8s-conformance/pull/1849>, the logo
<https://drive.google.com/file/d/1qeDvcfTLmr93pHJ0VHzeTjgE6OGkeIJg/view>
format is SVG file. Thank you!

Best Regards,
Iris Chan 詹子陞

ASUS Cloud Corporation｜www.asuscloud.com
TEL:+886-2-28987477 ext.86002
Mobile:+886-926335108

CONFIDENTIAL NOTE:
The information contained in this email and any attachments thereto is
intended solely for the addressee or addressees and may contain information
that is privileged, confidential and exempt from disclosure under
applicable law.  If you are not an intended recipient, please delete the
message and any attachments thereto and notify the sender of misdelivery.
You are hereby notified that any dissemination, distribution or copying of
this communication is strictly prohibited.  All liability for viruses is
excluded to the fullest extent permitted by applicable law.



CNCF CI Bot ***@***.***> 於 2022年4月7日 週四 下午1:30寫道：

**Comment by taylorwaggoner** [comment]
Please update the logo to https://landscape.cncf.io/logos/asus-cloud-infra.svg which would resolve the error. This is the logo we currently have displayed for this product in the landscape. Thanks!

**Comment by IrisChanTW** [comment]
Hi Taylor,

Thanks to your suggestion, we have updated the file and got a response from
the robot's feedback "All requirements (13) have passed for the submission!"
Thank you!

Best Regards,
Iris Chan 詹子陞

ASUS Cloud Corporation｜www.asuscloud.com
TEL:+886-2-28987477 ext.86002
Mobile:+886-926335108

CONFIDENTIAL NOTE:
The information contained in this email and any attachments thereto is
intended solely for the addressee or addressees and may contain information
that is privileged, confidential and exempt from disclosure under
applicable law.  If you are not an intended recipient, please delete the
message and any attachments thereto and notify the sender of misdelivery.
You are hereby notified that any dissemination, distribution or copying of
this communication is strictly prohibited.  All liability for viruses is
excluded to the fullest extent permitted by applicable law.


Taylor Waggoner ***@***.***> 於 2022年4月8日 週五 上午4:55寫道：

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Pull request merged**